libtransact/src/workload/runner.rs

```diff
@@ -0,0 +1,425 @@
+/*
+ * Copyright 2021 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------------
+ */
+
+//! Provides a runner to submit `BatchWorkload`s
+
+// TODO add builder for Worker
+
+use std::collections::HashMap;
+use std::fmt;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::mpsc::{channel, Sender, TryRecvError};
+use std::{thread, time};
+
+use reqwest::{blocking::Client, header, StatusCode};
+
+use crate::protos::IntoBytes;
+
+use super::error::WorkloadRunnerError;
+use super::BatchWorkload;
+
+const DEFAULT_LOG_TIME: u32 = 5; // time in seconds
```

Comment:
Would be helpful to add `_SECS` to the const name so you don't have to see this comment to know what it represents.

Reply:
updated